tests/filter/filter_plugin_test.py

```diff
@@ -32,11 +35,37 @@ def test_slow():
         '--tags=@tag1',
         '--tags=~@tag3',
         '-v',
-        '--zephyr-base=.'
+        '--zephyr-base=.',
+        extra_args
+    )
+    assert result.ret == 0
+    result.assert_outcomes(passed=2, failed=0, skipped=0)
+    result.stdout.fnmatch_lines([
```

Comment:
check also if other test are not matched:
result.stdout.no_fnmatch_line('*test_slow*')